src/IRPrinter.cpp

```diff
@@ -328,6 +328,11 @@ std::ostream &operator<<(std::ostream &stream, const LinkageType &type) {
     return stream;
 }
 
+std::ostream &operator<<(std::ostream &stream, const Indentation &indentation) {
+    for (int i = 0; i < indentation.indent; i++) stream << ' ';
```

Comment:
Nit: for for loops, I think we use braces and multiple lines.

Reply:
Will fix, but look at the deleted version of do_indent(), from which this was directly copied :-/

Reply:
Yeah, I noticed that. I spent some time yesterday trying to wrangle a clang-format file that I could blindly apply, but it's still the case that it fixes trivial things like this that aren't a big deal for readability, and then makes a horrible mess in other cases.